policyengine_us/parameters/gov/states/nh/tax/income/exemptions/amount/base.yaml

```diff
@@ -0,0 +1,12 @@
+description: New Hampshire provides this base exemption.
+metadata:
+  unit: currency-USD
+  label: New Hampshire base exemption amount
+  reference: 
+    - title: TITLE V - TAXATION - CHAPTER 77 - TAXATION OF INCOMES - Section 77:5
+      href: https://www.gencourt.state.nh.us/rsa/html/V/77/77-5.htm
+    - title: 2022 DP-10 Interest and Dividends Tax Return Instructions
+      href: https://www.revenue.nh.gov/forms/2023/documents/dp-10-instructions-2022.pdf
+values:
```

Comment:
Please add a breakdown by filing status. The form indicates it's $2,400 for all except joint which is $4,800.